spec/integration/hawkular_client_spec.rb

```diff
@@ -295,8 +349,8 @@ module Hawkular::Client::RSpec
         end
       end
 
-      xit 'Should both reuse the websocket connection', :websocket do
-        WebSocketVCR.record(example, self) do
+      it 'Should both reuse the websocket connection', :websocket do
```

Comment:
`it 'reuses the WebSocket connection'` would be a better name for this test. Also, it is not testing anything.